.github/actions/buildkite/README.md

```diff
@@ -0,0 +1,62 @@
+## About
+
+GitHub Action to run a BuildKite pipeline using Vault
+
+___
+
+* [Usage](#usage)
+  * [Configuration](#configuration)
+* [Customizing](#customizing)
+  * [inputs](#inputs)
+
+## Usage
+
+### Configuration
+
+
+```yaml
+---
+name: Run In BuildKite
+on:
+  workflow_run:
+    workflows: [ ci ]
```

Comment:
This more of a general comment.

Which syntax do we prefer as a team and want to be used?

```yaml
workflows:
  - ci
```

or 

```yaml
workflows: [ ci ]
```

I personally think the first approach is less error prone when adding or removing elements. But it's just a tiny thing.

Reply:
No strong opinion

```suggestion
    workflows:
      - ci
```

Reply:
Yeah, I guess this is just nitpicking.

But since, this is the documentation, future implementations might copy this style.